sys-apps/smc-sum/smc-sum-2.2.0.20190220-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="${PN/smc-/}"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="Updates the BIOS and IPMI firmware and system settings on Supermicro mainboards"
+HOMEPAGE="https://www.supermicro.com"
+SRC_URI="${MY_PN}_${MY_PV}_Linux_x86_64_${MY_DATE}.tar.gz"
+
+LICENSE="supermicro"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="module"
+
+RDEPEND="module? ( sys-apps/smc-sum-driver )"
```

Comment:
This doesn't seem to have any function in this ebuild? 
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs

Make it `optfeature` instead please.

Reply:
I disagree in this case. ```sum``` needs the kernel module for full features, when using it locally on super micro hardware. You only don't need it, when using it for remote access. So I don't think, ```optfeature```` is a good idea here.

Reply:
@gentoo/qa what do you think?

Not like this package altogether can be tested, 
```
Fetch instructions for sys-apps/smc-sum-driver-1.6.0.20151201-r2:
 * Please contact the Supermicro support at support@supermicro.com,
 * in order to obtain a copy of sum_driver_V1.6.0_20151201.tar.gz
 * and place it in your DISTDIR directory.
```

Reply:
I can provide you the source code, if you want.

Reply:
I wonder if their license allows that ;) if yes, you can send it to juippis@gmail or gentoo address,
Either way there are few things: I wanna hear from @mgorny what to do with this, I'm honestly wondering HOW this even got to the tree and should it still remain there - although there are several game-ebuilds with similar restrictions, so it might not be so uncommon...
and secondly you need to drop ~x86 from the reverse dep, preferably in a commit before it gets dropped from smc-sum.

Reply:
Send you the email. Basically, the driver could stay ~x86, as it can compile there.

Reply:
Didnt receive it :( not even in my spam folder. Maybe server-side declined... :\
Also I now see that smc-sum-driver isn't depending on this, so indeed the keywords don't break dependencies (I guess).

Reply:
Ok, send instead to gmail now.

Reply:
QA reply in IRC, it's ok. Thanks!

Reply:
Thanks :-)